docs/com.bitwarden.organizations.md

```diff
@@ -0,0 +1,79 @@
+[Table of contents](README.md#table-of-contents)
+
+# Bitwarden organization
+
+The `com.bitwarden.organization` doctype is used to group
+[ciphers](./com.bitwarden.ciphers.md) in a single unit and/or to share them with
+other [bitwarden contacts](./com.bitwarden.contacts.md).
+
+## Attributes
+
+- `name`: {string} - the name of the organization
+- `members`: {object} - the members of the organization (see below)
+- `defaultCollection`: {object} - the collection of the organization (see below)
+- `cozyMetadata`: {object} - the [cozyMetadata](README.md#document-metadata)
+
+## Members
+
+An organization can be shared with a list of members.
+
+- `email`: {string} - the email of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `key`: {string} - the public key of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `name`: {string} - the name of the member
+- `owner`: {bool} - true if the member owns the organization
+- `status`: {int} - the status of the member (`0` = `invited`, `1` = `accepted`, `2` = `confirmed`)
+- `user_id`: {string} - the id of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
```

Comment:
Here we duplicate the information from the `com.bitwarden.contacts`, and add a reference to it with the `user_id`: shouldn't we use a relationship to the contact instead?

Reply:
No, the `com.bitwarden.contacts` are not replicated by the sharings, and we need to send the `key` to the concerned member so that they can decrypt the ciphers.